openedx/adg/lms/applications/constants.py

```diff
@@ -118,3 +118,18 @@
 
 PROGRAM_PREREQ_COURSES_TITLE = f'<strong>{_("Program Prerequisite Courses:")}</strong>'
 BU_PREREQ_COURSES_TITLE = f'<strong>{_("Business Unit Prerequisite Courses:")}</strong>'
+
+APPLICATION_WAITLISTED_CONGRATS = _(
+    'Hey {username}, Thank you for visiting again. Your application is still under review.'
+)
+APPLICATION_WAITLISTED_INSTRUCTION = _(
+    'Admin has waitlisted your application. Please come back and revisit again in sometime to check your application '
+    'status.'
```

Comment:
Line break can be improved, We should break the first line a bit earlier to enhance readability.